runners/google-cloud-dataflow-java/worker/src/main/java/org/apache/beam/runners/dataflow/worker/StreamingStepMetricsContainer.java

```diff
@@ -81,6 +98,28 @@ protected StreamingStepMetricsContainer createContainer(String stepName) {
     };
   }
 
+  // Testing constructor.
```

Comment:
What does "Testing constructor" mean? I understand it as having some mock objects so one can test. Probably (Construct a ... that supports mock Clock, etc. For testing purposes only)